ICSharpCode.Decompiler.Tests/CorrectnessTestRunner.cs

```diff
@@ -486,7 +486,7 @@ async Task RunIL(string testFileName, CompilerOptions options = CompilerOptions.
 			bool optionsForce32Bit = options.HasFlag(CompilerOptions.Force32Bit);
 			bool asmOptionsForce32Bit = asmOptions.HasFlag(AssemblerOptions.Force32Bit);
 
-			Assert.AreEqual(optionsForce32Bit, asmOptionsForce32Bit, "Inconsistent architecture.");
+			Assert.That(asmOptionsForce32Bit, Is.EqualTo(optionsForce32Bit), "Inconsistent architecture.");
```

Comment:
I really don't like that overly verbose syntax.... what's wrong with `AreEqual`?

Reply:
https://docs.nunit.org/articles/nunit/writing-tests/assertions/assertions.html... why not keep the classic model for all the old tests?

Reply:
Guess what will happen if we keep classic. It will be copy pasted forever.

Reply:
I see no problem with that... :D the short forms are more intuitive...